### ISSUE TITLE
issue about training on zju_mocap_refine

### ISSUE THREAD

**Issue by HandSomeWay** [zju3dv/RelightableAvatar#6]
at first, thank you for your amazing work! 
i‘m trying to train avatar using ZJU_MOCAP_REFINE dataset ,and i ran the command
`python train.py  -c configs/my_zju_mocap/my_392_4v_geo.yaml`
then i got error 
![image](https://github.com/user-attachments/assets/5b8f0851-aeaf-4c35-93a4-fb668269d789)

how can i solve this problem

**Comment by lord-voldemrt** [comment]
Did you manage to resolve this issue? I encountered the same error while working with the synthetic human dataset.

**Comment by HandSomeWay** [comment]
Hello, sida, could you provide the document for training? i need to train my own dataset on this work, please. @pengsida

**Comment by HandSomeWay** [comment]
> Did you manage to resolve this issue? I encountered the same error while working with the synthetic human dataset.

Yes , you can change the `body_model: easymocap/output-output-smpl-3d/cfg_model.yml` into `body_model: easymocap/output-smplh-3d/cfg_model.yml` in configs/base.yaml

**Comment by HandSomeWay** [comment]
> Did you manage to resolve this issue? I encountered the same error while working with the synthetic human dataset.

but i can't train into relightable stage, do you know how to do it?
